services/processor/native/call.go

```diff
@@ -77,45 +84,51 @@ func (s *service) processMethodCall(contractInfo *types.ContractInfo, methodInfo
 	}
 
 	// get contract error
-	outErr, err := s.createContractError(methodInfo, outValues[len(outValues)-1])
+	outErr, err = s.createContractError(methodInfo, outValues[len(outValues)-1])
 	return outArgs, outErr, err
 }
 
-func (s *service) verifyMethodArgs(methodInfo *types.MethodInfo, implementation interface{}, args []*protocol.MethodArgument) ([]reflect.Value, error) {
+func (s *service) verifyMethodArgs(ctx types.Context, methodInfo *types.MethodInfo, implementation interface{}, args []*protocol.MethodArgument) ([]reflect.Value, error) {
```

Comment:
The name is a bit misleading. Seems like it does more than just verify the actual arguments against the expected method params, otherwise it would've returned only a bool

Reply:
fixed